web/src/condense.js

```diff
@@ -21,11 +21,13 @@ This library implements two related, similar concepts:
 const _message_content_height_cache = new Map();
 
 function show_more_link($row) {
+    $row.find(".message_length_controller").show();
     $row.find(".message_condenser").hide();
     $row.find(".message_expander").show();
 }
 
 function show_condense_link($row) {
+    $row.find(".message_length_controller").show();
     $row.find(".message_expander").hide();
     $row.find(".message_condenser").show();
```

Comment:
Are these changes needed? I'd expect `message_length_controller` to be always visible.

Reply:
Upon going through the implementation again, these changes weren't needed and there was a simpler solution, which I have now pushed to the PR, thanks!